src/views/About.vue

```diff
@@ -188,6 +189,7 @@ img {
     max-width: 50%;  /* Sets the maximum width to 50% of its container */
     height: auto;    /* Maintains the aspect ratio of the image */
 }
-
-  </style>
-  
\ No newline at end of file
+.top-image {
+    max-width: 100% !important;  /* Set the maximum width to 100% of its container */
```

Comment:
Using important is okay in this situation but often other styling choices should be attempted to be used first